packages/shared/lib/stronghold.ts

```diff
@@ -36,12 +36,12 @@ export const checkStronghold = (callback: any): void => {
             } else {
                 callback()
             }
-        },
-        onError(err) {
+        })
+        .catch((err) => {
+            console.error(err)
```

Comment:
Do we accept `console.log`s on dev?

Reply:
no console logs, just console.error. We still need to agree on the error handling app wide.